tests/deployd/test_common.py

```diff
@@ -140,7 +140,7 @@ def test_get_service_instances_needing_update():
         ]
         mock_load_marathon_service_config.side_effect = mock_configs
         ret = get_service_instances_needing_update(mock.Mock(), mock_service_instances, 'westeros-prod')
-        assert ret == [('universe', 'c137'), ('universe', 'c138')]
+        assert ret == [('universe', 'c138')]
 
         mock_configs = [
             mock.Mock(format_marathon_app_dict=mock.Mock(side_effect=InvalidJobNameError)),
```

Comment:
Can you dupe this stanza with `NoDeploymentsAvailable` as the side_effect please. Just so we cover all the exceptions in the test.